deploy/crds/devconsole_v1alpha1_envdeployment_crd.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: envdeployments.devconsole.openshift.io
+  name: envdeployments.devopsconsole-operator.openshift.io
 spec:
-  group: devconsole.openshift.io
+  group: devopsconsole-operator.openshift.io
```

Comment:
I think we should use `devopsconsole.openshift.io` group for all our DevOps Console APIs. Not 'devopsconsole-operator.openshift.io'.

Reply:
Ah right! https://github.com/redhat-developer/devopsconsole-operator/pull/4/commits/d65f2374124b477a156a669c966561b240929bc3

Reply:
Looks good. What about the other places too? `devopsconsole-operator.openshift.io` group is all over the place :)

Reply:
Right https://github.com/redhat-developer/devopsconsole-operator/pull/4/commits/cf023c28b7b7cb31d4302a7e5c7f02da151ae9c0